### DISCUSSION TITLE
Can i change the Magisk to Magisk Alpha?

### DISCUSSION THREAD

**Discussion by krisn7** [LSPosed/MagiskOnWSALocal#653] [Q&A] [answered]
how or can you change Magisk original ver to Alpha version or packing the Alpha ver instead of Original. If you can please teel me how

**Comment by fromhgbwithluv** [comment, marked as answer]
You can, but you need mannual download Magisk Alpha, WSA, MindTheGapps.
So I think you should use my repos to create Custom Build (read the README.md for guide)
https://github.com/YT-Advanced/WSA-Script

**Reply by krisn7** [reply]
OMG yes custom builds im realize now. Thank youuu <3
